custom_components/zwave_mqtt/entity.py

```diff
@@ -189,7 +189,7 @@ def device_info(self):
         }
         # device with multiple instances is split up into virtual devices for each instance
         if node_instance > 1:
-            parent_dev_id = f"{ozw_instance}.{node.node_id}.1"
+            parent_dev_id = create_device_id(node, 1)
```

Comment:
1 is the default for node_instance.
```suggestion
            parent_dev_id = create_device_id(node)
```